.drone.star

```diff
@@ -213,6 +215,7 @@ def apiTests(ctx, coreBranch = 'master', coreCommit = ''):
         ] + ([
           'git checkout %s' % (coreCommit)
         ] if coreCommit != '' else []) + [
+          '/drone/src/tests/file_descriptor.sh &',
```

Comment:
This is running in the docker of the test-runner.
Should it be running in the docker of the ocis server-under-test?